packages/main/src/webComponents/IllustratedMessage/index.tsx

```diff
@@ -160,6 +160,15 @@ export interface IllustratedMessagePropTypes extends IllustratedMessageAttribute
    * Learn more about it [here](https://sap.github.io/ui5-webcomponents-react/?path=/docs/knowledge-base-handling-slots--page).
    */
   subtitle?: ReactNode;
+  /**
+   * Defines the title of the component.
+   *
+   * **Note:** Using this slot, the default title text of illustration and the value of `title` property will be overwritten.
+   *
+   * __Note:__ When passing a custom React component to this prop, you have to make sure your component reads the `slot` prop and appends it to the most outer element of your component.
+   * Learn more about it [here](https://sap.github.io/ui5-webcomponents-react/?path=/docs/knowledge-base-handling-slots--page).
+   */
+  title?: ReactNode;
```

Comment:
This must be removed, we don't support the `title` slot for the IllustratedMessage.